crates/stc_ts_type_checker/tests/conformance/types/keyof/keyofAndIndexedAccess.error-diff.json

```diff
@@ -11,6 +11,7 @@
     "TS2345": [
       96,
       132,
+      162,
```

Comment:
when call JsFunction, we overwrite generic

```ts

function f33<S extends Shape, K extends keyof S>(shape: S, key: K) {
    let name = getProperty(shape, "name");
    let prop = getProperty(shape, key);
    return prop;
}

function f34(ts: TaggedShape) {
    let tag1 = f33(ts, "tag");
    let tag2 = getProperty(ts, "tag");
}
```

Change S to TaggedSchema, K to "tag"